frontend/src/routes/+layout.svelte

```diff
@@ -2,21 +2,50 @@
   import "../app.scss";
 </script>
 
-<div class="columns m-6">
-  <div class="column is-one-third" />
-  <div class="column is-one-third">
-    <div class="level">
-      <div class="level-item">
-        <h1 class="title is-0">Rational Trading</h1>
-      </div>
+<!-- the navbar -->
+<nav class="navbar is-primary">
+  <div class="navbar-brand">
+    <div class="navbar-item">
+      <img src="/logo.png" width="177" height="50" alt="logo" />
+    </div>
+
+    <!-- mobile burger ignore for now
+    <a
+      role="button"
+      class="navbar-burger"
+      aria-label="menu"
+      aria-expanded="false"
+      data-target="navbarBasicExample"
+    >
+      <span aria-hidden="true" />
+      <span aria-hidden="true" />
+      <span aria-hidden="true" />
+    </a>
+  -->
+  </div>
+
+  <div class="navbar-menu">
+    <div class="navbar-start">
+      <a class="navbar-item" href="/"> Home </a>
+
+      <a class="navbar-item" href="/explore"> Explore </a>
+
+      <a class="navbar-item" href="/portfolio"> Portfolio </a>
     </div>
-    <div class="level">
-      <div class="level-item"><a href="/">Home</a></div>
-      <div class="level-item"><a href="/shopping">Shopping</a></div>
+
+    <div class="navbar-end">
+      <div class="navbar-item">
+        <div class="buttons">
+          <button class="button is-info">
```

Comment:
I would leave these as links for now (as modals can be a bit tricky) but up to you.